### PR TITLE
bug/308_309_305

### PR DESCRIPTION
**Pull request by frbattid**
Fixes issues https://github.com/telefonicaid/fiware-connectors/issues/305 (@vgarciag , please, LGTM it)
Fixes issues https://github.com/telefonicaid/fiware-connectors/issues/308 and https://github.com/telefonicaid/fiware-connectors/issues/309 (@mcarracedo , please, LGTM it)

This PR must be twined in order to merge its content within `develop` as well.

### CODE REVIEW

**Comment by mcarracedo** [comment]
LGTM

File: flume/README.md [hunk 1]
```diff
@@ -398,6 +398,7 @@ cygnusagent.sinks.hdfs-sink.channel = hdfs-channel
 # sink class, must not be changed
```
Comment (fgalan) [inline flume/README.md]: CHANGES_NEXT_RELEASE entries would be needed for #305, #308 and #309 ?

Comment (frbattid) [inline flume/README.md]: Fixed in https://github.com/telefonicaid/fiware-connectors/commit/e820c273f2a3b1bbac288b3c750d48144d7572c8

**Comment by fgalan** [comment]
LGTM

**Comment by vgarciag** [comment]
LGTM

**Pull request merged**